### ISSUE TITLE
save() output advice: usepgfplotslibrary

### ISSUE THREAD

**Issue by homocomputeris** [nschloe/matplotlib2tikz#177]
`save()` function advises:
```
Please add the following lines to your LaTeX preamble:
\usepackage[utf8]{inputenc}
\usepackage{fontspec} % This line only for XeLaTeX and LuaLaTeX
\usepackage{pgfplots}
\usepgfplotslibrary{plotmarks}
```
However, pdfLaTeX casts an error:
```
I can't find file `pgflibrarypgfplots.plotmarks.code.tex'. \usepgfplotslibrary{plotmarks}
Emergency stop. \usepgfplotslibrary{plotmarks}
```

But with `\usetikzlibrary{plotmarks}` it works fine.
Should it be updated?

**Comment by nschloe** [comment]
Fixed via 0a97368fd44df3bb988fca0d51cd9301ea8766bd.

**Comment by homocomputeris** [comment]
Thanks for fixing.

How can I disable or suppress this output?
